org.eclipse.jdt.ls.filesystem/src/org/eclipse/jdt/ls/core/internal/filesystem/JDTLSFilesystemActivator.java

```diff
@@ -1,5 +1,14 @@
 package org.eclipse.jdt.ls.core.internal.filesystem;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+import org.eclipse.core.runtime.preferences.IEclipsePreferences;
```

Comment:
You have some unused imports here. `IEclipsePreferences` and `InstanceScope`.

Reply:
Fixed.